test/abstractarray.jl

```diff
@@ -273,3 +273,29 @@ end
     @test @inferred(hcat(SA[1,2,3])) === SMatrix{3,1}(1,2,3)
     @test @inferred(hcat(SA[1 2 3])) === SA[1 2 3]
 end
+
+if VERSION >= v"1.6.0-DEV.1334"
```

Comment:
```suggestion
@static if VERSION >= v"1.6.0-DEV.1334"
```
This should prevent failures on older versions.